### PR TITLE
update api docs

### PR DESCRIPTION
**Pull request by jenpaff**

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -351,30 +347,84 @@ components:
...
+      description: "An optional status filter. Multiple statuses can be requested."
+      enum: ["created", "deposited", "pending", "active", "exited", "withdrawn"]
     ValidatorsRequest:
       type: "object"
       properties:
         from:
-          type: string
-          format: date
-          example: "2019-05-17"
           description: |
```
Comment (QAston) [inline swagger.yaml:381]: while this is technically correct change, we cannot use this because the code generator doesn't support oneOf/anyOf/allOf

Comment (jenpaff) [reply swagger.yaml:381]: how so? seemed to generate fine for me

Comment (QAston) [reply swagger.yaml:381]: the type checking doesn't work properly, and the generator lists these as unsupported https://openapi-generator.tech/docs/generators/rust#schema-support-feature

Comment (QAston) [reply swagger.yaml:381]: we looks at this with @mksh before and decided to keep the spec this way

**Review by QAston — APPROVED** [review]

**Pull request merged**